apps/arweave/src/ar.erl

```diff
@@ -178,7 +178,7 @@ parse_cli_args(["init"|Rest], C) ->
 parse_cli_args(["mine"|Rest], C) ->
 	parse_cli_args(Rest, C#config { mine = true });
 parse_cli_args(["peer", Peer|Rest], C = #config { peers = Ps }) ->
-	parse_cli_args(Rest, C#config { peers = [ar_util:parse_peer(Peer)|Ps] });
+	parse_cli_args(Rest, C#config { peers = [ar_util:safe_parse_peer(Peer)|Ps] });
```

Comment:
`ar_util:safe_parse_peer/1` will return `{ok, Peer}` instead `Peer` - looks like by this reason you have a crash in tests